charts/soroban-rpc/values-pubnet.yaml

```diff
@@ -201,12 +201,12 @@ sorobanRpc:
   persistence:
     enabled: false
     storageClass: default
-    size: 100G
+    size: 300G
```

Comment:
Any specific reason for increasing the storage here? Our current statefulset had 200G right?

Reply:
yes, saw directory space usage on a full 24hr populated rpc pod, being over 100gb total between rpc and captive core files.